inst/examples/002_more_inputs/app.R

```diff
@@ -32,84 +31,79 @@ server <- function(input, output, session) {
-          textOutput("textOutput9"),
-          h4("Dropdown"),
-          Dropdown("input10", "A", options = options),
-          textOutput("textOutput10"),
-          h4("DatePicker"),
-          DatePicker("input11", "2020-06-25T22:00:00.000Z", strings = dayPickerStrings),
-          textOutput("textOutput11"),
-          h4("SwatchColorPicker"),
-          SwatchColorPicker("input12", "orange", colorCells = colorCells, columnCount = length(colorCells)),
-          textOutput("textOutput12"),
-          h4("Toggle"),
-          Toggle("input13", TRUE),
-          textOutput("textOutput13"),
-          h4("SearchBox"),
-          SearchBox("input14", placeholder = "Search"),
-          textOutput("textOutput14"),
-        )
+      div(
+        h4("Checkbox"),
+        Checkbox.i("value1", value = FALSE),
+        textOutput("text1"),
+        h4("Rating"),
+        Rating.i("value2", value = 2),
+        textOutput("text2"),
+        h4("Slider"),
+        Slider.i("value3", value = 42, min = -100, max = 100),
+        textOutput("text3"),
+        h4("TextField"),
+        TextField.i("value4"),
+        textOutput("text4"),
+        h4("SpinButton"),
+        SpinButton.i("value5", value = 15, min = 0, max = 50, step = 5),
+        textOutput("text5"),
+        h4("Calendar"),
+        Calendar.i("value6", value = "2020-06-25T22:00:00.000Z", strings = dayPickerStrings),
+        textOutput("text6"),
+        h4("ChoiceGroup"),
+        ChoiceGroup.i("value7", value = "B", options = options),
+        textOutput("text7"),
+        h4("ColorPicker"),
+        ColorPicker.i("value8", value = "#00FF01"),
+        textOutput("text8"),
+        h4("ComboBox"),
+        ComboBox.i("value9", value = list(text = "some text"), options = options, allowFreeform = TRUE),
+        textOutput("text9"),
+        h4("Dropdown"),
+        Dropdown.i("value10", value = "A", options = options),
+        textOutput("text10"),
+        h4("DatePicker"),
+        DatePicker.i("value11", value = "2020-06-25T22:00:00.000Z", strings = dayPickerStrings),
+        textOutput("text11"),
+        h4("SwatchColorPicker"),
+        SwatchColorPicker.i("value12", value = "orange", colorCells = colorCells, columnCount = length(colorCells)),
+        textOutput("text12"),
+        h4("Toggle"),
+        Toggle.i("value13", value = TRUE),
+        textOutput("text13"),
+        h4("SearchBox"),
+        SearchBox.i("value14", placeholder = "Search"),
+        textOutput("text14"),
       )
     } else {
       NULL
     }
   })
 
-  lapply(c(1:8, 10:14), function(i) {
-    inputName <- paste0("input", i)
-    outputName <- paste0("textOutput", i)
+  lapply(1:14, function(i) {
+    inputName <- paste0("value", i)
+    outputName <- paste0("text", i)
     output[[outputName]] <- renderText({
-      sprintf("Value: %s", input[[inputName]])
+      value <- capture.output(dput(input[[inputName]]))
```

Comment:
It is not needed, but `dput()` provides more useful output for this example. For value being `NULL`, `FALSE`, or `"FALSE"` we get `value: NULL`, `value: FALSE` and `value: "FALSE"` respectively - instead of empty string in the first case and `value: FALSE` in the other two cases.

Unfortunately, `dput()` can only print to file / standard output - that's what the `capture.output` is for. I extracted this into a function.